pkgs/applications/version-management/bit/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "bit";
+  version = "0.5.11";
+
+  src = fetchFromGitHub {
+    owner = "chriswalz";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1rdi6wsr5w64gvqkdysxrxfjxi3xbwigy3sk8wsx9zp4fql9xnw1";
+  };
+
+  subPackages = [ "." ];
```

Comment:
Necessary?

Reply:
Maybe not, pulled from another example, squashes a branch testing bug.